src/tmpfiles/tmpfiles.c

```diff
@@ -2487,6 +2483,11 @@ static int patch_var_run(const char *fname, unsigned line, char **path) {
         return 0;
 }
 
+static void deprecate_for_force(const char from, const char to, const char *fname, const int line) {
+        assert(fname);
+        log_warning("%c is deprecated; use %c+ instead: %s:%d", from, to, fname, line);
```

Comment:
Please remove the warning. We should only emit a warning if something is problematic, but in this case it is totally OK to keep using `F` in existing code. If we emit a warning, people have to adjust or suffer the warning, so we should reserve this for cases where it is important to update.

Reply:
Ok. I've set the log level to `notice`. I found a similar message [here](https://github.com/systemd/systemd/blob/master/src/core/load-fragment.c#L3874). Let me know if you think it should be different.

Reply:
Yes, please just remove the warning completely.